packages/composer/src/components/Footer/index.tsx

```diff
@@ -28,12 +28,18 @@ export default function Footer() {
       {tips.map(tip => (
         <Tip key={tip.name || tip.combo}>
           <Mark>
-            {tip.combo.split('+').map((symbol, i) => (
-              <span key={symbol}>
-                {i !== 0 && ' + '}
-                <Symbol char={SYMBOLS[symbol] || symbol.toLowerCase()} />
-              </span>
-            ))}
+            {tip.combo.split('+').map((symbol, i) => {
```

Comment:
Could use `.filter(symbol => !symbol.startsWith('!'))` instead of `if () { return null }`